security/responding.md

```diff
@@ -0,0 +1,143 @@
+# Vulnerability Discolure Response Policy
+
+The project follows the following process when responding to vulnerabilities
+raised via the Security Disclosure Process.
+
+## Product Security Taskforce Membership
+
+The subset of maintainers who are subscribed to the security@knative.team email
+address form the Product Security Taskforce. 
```

Comment:
why PST rather than VMT?

Reply:
PST is the form of words used in the envoy doc. I'm totally open to either if you feel strongly VMT is better.